entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
I'm on Windows 10 and getting this message below

`docker-compose up --build`

```
Creating fastapi-heroku_web_1 ... done
Attaching to fastapi-heroku_web_1
web_1  | standard_init_linux.go:228: exec user process caused: no such file or directory
fastapi-heroku_web_1 exited with code 1
```

Btw I tried on Ubuntu, it seems ok.

Reply:
I think it's about CRLF -> LF
https://stackoverflow.com/a/52665687/7030591

Reply:
https://stackoverflow.com/a/19283453/7030591